install.audius.org.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+curl -sSL "https://github.com/AudiusProject/audius-d/releases/latest/download/audius-ctl-$(uname -m)" -o ~/.local/bin/audius-ctl
+chmod +x ~/.local/bin/audius-ctl
+echo "audius-ctl has been installed to ~/.local/bin/audius-ctl"
+echo "You can run it using: $ audius-ctl"
```

Comment:
This does get a bit tricky because I don't think the directory ~/.local/bin exists by default, and I also don't think it's automatically in $PATH. We may need to either 1. ask for sudo permissions to install to /usr/loca/bin or 2. mkdir -p ~/.local/bin and add instructions for updating PATH (or 3. create a homebrew/apt package or something).